packages/fluentui/react-northstar-fela-renderer/src/createFelaRenderer.tsx

```diff
@@ -46,9 +46,17 @@ if (process.env.NODE_ENV !== 'production' && process.env.NODE_ENV !== 'test') {
   }
 }
 
-// Blacklist contains a list of classNames that are used by FontAwesome
-// https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use
-const blacklistedClassNames = ['fa', 'fas', 'far', 'fal', 'fab'];
+const blacklistedClassNames = [
+  // Blacklist contains a list of classNames that are used by FontAwesome
+  // https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use
+  'fa',
+  'fas',
+  'far',
+  'fal',
+  'fab',
+  // .cke is used by CKEditor
+  'cke',
```

Comment:
https://ckeditor.com/docs/ckeditor5/latest/examples/builds/classic-editor.html

v5 seems to be able to use `ck`

```suggestion
  'cke',
  'ck',
```

Reply:
Yup, good catch 👍